src/components/BookList.js

```diff
@@ -0,0 +1,32 @@
+import React from 'react';
+import Books from './Books';
+
+const BookList = () => {
+  const mockBooks = [
+    {
+      id: '0',
+      title: 'Book-1',
+      author: 'author-1',
+    },
+    {
+      id: '1',
+      title: 'Book-2',
+      author: 'author-2',
+    },
+  ];
+  return (
+    <ul>
+      {
+      mockBooks.map((book) => (
+        <Books
+          key={book.id}
+          title={book.title}
+          author={book.author}
+        />
+      ))
+    }
+    </ul>
```

Comment:
- Hint: The bookList component is the only component inside the main page, and this doesn't have the form component. Now, imagine that the form is inside the Book component, creating a form for every book, so it couldn't be there.

Reply:
it seems like i'll have to repeat the week, then...